build-a-game-playing-agent/agent_test.py

```diff
@@ -21,7 +21,6 @@
 from queue import Empty as QueueEmptyError
 from importlib import reload
 
-
```

Comment:
hehehehe TOP-LEVEL (lexical scope, not declaration order) function and class *definitions* should be _surrounded_ with a blank like... _not_ the member methods/attributes.